parm/use_cases/met_tool_wrapper/GridStat/GridStat.conf

```diff
@@ -129,6 +129,7 @@ GRID_STAT_OUTPUT_PREFIX = {MODEL}_{CURRENT_FCST_NAME}_vs_{OBTYPE}_{CURRENT_OBS_N
 #GRID_STAT_CLIMO_CDF_BINS = 1
 #GRID_STAT_CLIMO_CDF_CENTER_BINS = False
 #GRID_STAT_CLIMO_CDF_WRITE_BINS = True
+#GRID_STAT_CLIMO_CDF_DIRECT_PROB =
```

Comment:
Why do we have commented-out defaults listed for all the other CLIMO_CDF options, but no the new DIRECT_PROB one?

Reply:
We should come up with a standard for handling this when we clean up all of the config files. I'm not sure if we have an issue for that, but we would not do that until the next development cycle (5.0.0). When I add support for a new config variables,  I put a value so I can test setting the variable to see if it works with the MET app. After testing, I have been removing the value and comment out the variable. When I first added support for these other climo_cdf variables, I was leaving the value so users could see what type of value is expected for these variables. It would be good to be consistent but either way, the purpose of these comments variables are to show examples of all of the config variables that can be used in the basic use case.